graph-data.rs/Cargo.toml

```diff
@@ -5,7 +5,7 @@ authors = ["Vadim Rutkovsky <vrutkovs@redhat.com>"]
 edition = "2018"
 
 [dependencies]
-cincinnati = { git = "https://github.com/openshift/cincinnati", rev = "67fe062a9e4b9df11c014b4c7fcfd3d8179b508a"}
+cincinnati = { git = "https://github.com/hongkailiu/cincinnati", rev = "4c7aae63b686738e2be679f88589f4cddf31d91f"}
```

Comment:
This is for CI testing.

We will use https://github.com/openshift/cincinnati/pull/1009 instead.